src/test/AaveV2CollectorContractConsolidation.t.sol

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.17;
+
+import "@forge-std/Test.sol";
+import "@forge-std/console.sol";
+import {AaveV2Ethereum} from "@aave-address-book/AaveV2Ethereum.sol";
+import {AaveV2CollectorContractConsolidation} from "../AaveV2CollectorContractConsolidation.sol";
+import {AggregatorV3Interface} from "../external/AggregatorV3Interface.sol";
+import {ERC20} from "@openzeppelin/token/ERC20/ERC20.sol";
+
+contract AaveV2CollectorContractConsolidationTest is Test {
```

Comment:
Also have a test for some asset that's not initialized through the constructor. (i.e it should not work)

Reply:
Will do.